elifearticle/parse.py

```diff
@@ -145,6 +145,20 @@ def build_datasets(dataset_json):
     return datasets
 
 
+def build_data_availability(datasets_json):
+    """
+    Given datasets in JSON format, get the data availability from it if present
+    """
+    data_availability = None
+    if 'availability' in datasets_json:
+        # only expect one paragraph of text
+        try:
+            data_availability = datasets_json.get('availability')[0].get('text')
+        except IndexError:
```

Comment:
can a log or something similar be written here? Helps with diagnosing what is happening when the field is not being picked up

Reply:
I will refactor here to not include the exception. Exception handling is further up. This was just to account for some bad data if it occurred, and instead it will just only accept good data and ignore any other.